app/mailers/event_mailer.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class EventMailer < ApplicationMailer
+  before_action { @event = params[:event] }
+  before_action :set_emails
+
+  def monthly_donation_summary
+    @event = params[:event]
+
+    @donations = @event.donations.select { |donation| donation.created_at >= 1.month.ago }.sort_by(&:created_at)
```

Comment:
```suggestion
    @donations = @event.donations.where(created_at: Time.now.last_month.beginning_of_month..).order(:created_at)
```
I rewrote this in SQL so we can sum for L11